### PR TITLE
i18n(es): Update `styling.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `styling.mdx`
* [8f58358](https://github.com/withastro/docs/commit/8f583585da048de4286149c0fa9743181097e04b)
* [1864a2d](https://github.com/withastro/docs/commit/1864a2d045105b847d30bdf988564e0c7c7356aa)
* [5d05346](https://github.com/withastro/docs/commit/5d05346ada2fcc9250bfbc9f3ecfcfe7a55b03b9)
* [a916911](https://github.com/withastro/docs/commit/a916911d52ddd288213375a9e28e75da872998d3)
* [6e5daac](https://github.com/withastro/docs/commit/6e5daac325d8cd64f4f3748463c6ef272c582b72)

### CODE REVIEW

File: src/content/docs/es/guides/styling.mdx [hunk 1]
```diff
@@ -145,6 +160,15 @@ Este patrón te permite estilar componentes hijos directamente. Astro pasará el
 Debido a que la propiedad `class` incluye al hijo en el ámbito de su padre, es posible que los estilos caigan en cascada de padre a hijo. Para evitar que esto tenga efectos secundarios no deseados, asegúrese de usar nombres de clase únicos en el componente secundario.
 :::
 
+### Estilos en línea
+
+Puedes estilizar elementos HTML en línea utilizando el atributo `style`. Esto puede ser un string CSS o un objeto de propiedades CSS.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/styling.mdx:165]:
```suggestion
Puedes estilar elementos HTML en línea utilizando el atributo `style`. Esto puede ser un string CSS o un objeto de propiedades CSS.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo una sugerencias @Waxer59, buen trabajo ✅

**Comment by Waxer59** [comment]
Gracias por la revisión 😀

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**